providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/DiskStorageAccountType.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.azurecompute.arm.domain;
+
+/**
+ * DiskStorageAccountTypes used in Azure.
+ *
+ * @see <a href="https://docs.microsoft.com/en-us/rest/api/compute/disks/createorupdate#diskstorageaccounttypes">this page</a>
+ */
+public enum DiskStorageAccountType
```

Comment:
I've just seen we already had the [StorageAccountType](https://github.com/apache/jclouds/blob/master/providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/StorageAccountType.java) for disks. Can you add the missing values there instead? (Apologies for the noise, it's been quite some time since the last time I used the ARM provider :)).